pylatex/utils.py

```diff
@@ -249,54 +236,304 @@ def bold(s, *, escape=True):
+
+    return NoEscape(r'\newline ')
+
+
+def horizontal_fill():
+    r"""Fill the current line."""
+
+    return NoEscape(r'\hfill')
+
+
+def horizontal_skip(size):
+    r"""Add/remove the amount of horizontal space between elements.
+
+    Args
+    ----
+    size: str
+        The amount of horizontal space to add
+    """
+
+    return pylatex.base_classes.UnsafeCommand("hspace*", arguments=size)
+
+
+def display_page_number():
+    r"""Provide the page number in the following format: Page # of ##."""
+
+    return NoEscape(r'Page \thepage\ of \pageref{LastPage}')
+
+
+def huge(s, *, escape=True):
+    r"""Highlight the text as a header of size Huge.
+
+    Args
+    ----
+    s : str
+        The string to be formatted.
+    escape: bool
+        If true the enlarged text will be escaped
+    Returns
+    -------
+    NoEscape
+        The formatted string.
+    """
+
+    if escape:
+        s = escape_latex(s)
+
+    return NoEscape(r'\begin{Huge}' + s + '\end{Huge}')
+
+
+def header1(s, *, escape=True):
+    r"""Highlight the text as a header of size Large.
+
+    Args
+    ----
+    s : str
+        The string to be formatted.
+    escape: bool
+        If true the enlarged text will be escaped
+    Returns
+    -------
+    NoEscape
+        The formatted string.
+    """
+
+    if escape:
+        s = escape_latex(s)
+
+    return NoEscape(r'\begin{Large}' + s + '\end{Large}')
+
+
+def header2(s, *, escape=True):
+    r"""Highlight the text as a header of size large.
+
+    Args
+    ----
+    s : str
+        The string to be formatted.
+    escape: bool
+        If true the enlarged text will be escaped
+    Returns
+    -------
+    NoEscape
+        The formatted string.
+    """
+
+    if escape:
+        s = escape_latex(s)
+
+    return NoEscape(r'\begin{large}' + s + '\end{large}')
+
+
+def small1(s, *, escape=True):
+    r"""Highlight the text as size small.
+
+    Args
+    ----
+    s : str
+        The string to be formatted.
+    escape: bool
+        If true the enlarged text will be escaped
+    Returns
+    -------
+    NoEscape
+        The formatted string.
+    """
+
+    if escape:
+        s = escape_latex(s)
+
+    return NoEscape(r'\begin{small}' + s + '\end{small}')
+
+
+def small2(s, *, escape=True):
+    r"""Highlight the text as size footnotesize.
+
+    Args
+    ----
+    s : str
+        The string to be formatted.
+    escape: bool
+        If true the enlarged text will be escaped
+    Returns
+    -------
+    NoEscape
+        The formatted string.
+    """
+
+    if escape:
+        s = escape_latex(s)
+
+    return NoEscape(r'\begin{footnotesize}' + s + '\end{footnotesize}')
+
+
+def vertical_skip(size):
+    r"""Add the user specified amount of vertical space to the document.
+
+    Args
+    ----
+    size: str
+        The amount and units of vertical space to create
+    """
+
+    return pylatex.base_classes.UnsafeCommand("vspace*", arguments=size)
+
+
+def text_box(s):
```

Comment:
If it's not actually that useful (you can also remove it). I see you also added mdframed, which I think does something similar.